server/utils/FirebaseAlgoliaServer/functions/tests/cloudFunctionsIntegrationTests.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * BACKGROUND
+ * Integration tests for: 
+ * addEntityToIndex, deleteEntityFromIndex, updateRecordInIndex.
+ *
+ * These functions essentially link two different databases in realtime,
+ * And they cannot be directly tested because of how cloud functions work
+ * And because algolia does not provide a JS mocking library (AFAIK.)
+ * So testing has to be done in realtime online,
+ * Which makes testing somewhat difficult due to asynchronous operations
+ * Happening in weird orders.
+ * SO this code doesn't use unittests (couldn't get them to work), 
+ * and instead does live operations on the DBs, checks the DB state, 
```

Comment:
Are these running on prod DBs?  Don't mess with prod data in tests.  Make test DBs that mirror the settings of the prod DBs but are only used for tests.